src/main/java/org/broadinstitute/consent/http/resources/TDRResource.java

```diff
@@ -47,4 +47,21 @@ public Response getApprovedUsers(@Auth AuthUser authUser, @PathParam("identifier
             return createExceptionResponse(e);
         }
     }
+
+    @GET
+    @Produces("application/json")
+    @PermitAll
```

Comment:
is this the correct authentication scheme?

Reply:
Yes, exactly. This allows for any authenticated user to access the endpoint.